math/mathcore/inc/TRandom3.h

```diff
@@ -35,7 +35,7 @@ class TRandom3 : public TRandom {
    ~TRandom3() override;
    /// return current element of the state used for generate the random number
```

Comment:
I think we should specify clearly here what is return
Maybe something like: 

Return one element of the generator state used to generate the random numbers. 
Note that it is not the seed of the generator that was used in the SetSeed function and the full state (624 numbers) 
is required to define the generator and have a reproducible output.

Reply:
Yes that's ok with me. You may just modify my PR accordingly.